src/client/pages/SignInPage.tsx

```diff
@@ -1,6 +1,23 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { SignIn } from '@/client/pages/SignIn';
+import { ClientState } from '@/shared/model/ClientState';
+import { ClientStateContext } from '@/client/components/ClientState';
 
 export const SignInPage = () => {
-  return <SignIn />;
+  const clientState: ClientState = useContext(ClientStateContext);
+  const { pageData = {} } = clientState;
+  const { returnUrl, fieldErrors, email } = pageData;
+  const returnUrlQuery = returnUrl
+    ? `?returnUrl=${encodeURIComponent(returnUrl)}`
+    : '';
+
+  const errorSummary = fieldErrors?.find((e) => e.field === 'summary')?.message;
```

Comment:
Not for this PR, but it feels a bit wrong to read the first item from the array, ignoring the rest. This obviously works here as there's only one field but I wonder if this api could be made better?

Reply:
We've removed this line, the error now comes in the same way as the other errors.